scrapy/downloadermiddlewares/httpcompression.py

```diff
@@ -10,10 +10,13 @@
 ACCEPTED_ENCODINGS = [b'gzip', b'deflate']
 
 try:
+    flag=0
     import brotli
     ACCEPTED_ENCODINGS.append(b'br')
 except ImportError:
     pass
+else:
+    flag=1
 
```

Comment:
Just taking the chance here.
You could change `encoding = content_encoding.pop()` to `encoding = content_encoding[0]` in `process_response`, as this removes the header when it arrives in the spider and can make our life a bit hard to detect the issue.

Not sure if it follows here or as a fix at all.
cc @Gallaecio

Reply:
I see what you mean, [line 48](https://github.com/scrapy/scrapy/blob/35850f12207991eb815a5d39aaa7a5bc367e7d04/scrapy/downloadermiddlewares/httpcompression.py#L48), and yes, I think you are right.

I would not require this change in this pull request, but if not done here, we should probably open a ticket for it.